repos.d/rpm/amazonlinux.yaml

```diff
@@ -0,0 +1,33 @@
+- name: amzn2
+  type: repository
+  desc: Amazon Linux 2
+  family: fedora
+  minpackages: 2000
+  sources:
+    - name: core
+      fetcher: RepodataSqliteFetcher
+      parser: RepodataSqliteParser
+      url: http://amazonlinux.us-west-2.amazonaws.com/2/core/2.0/SRPMS/mirror.list
+  repolinks:
+    - desc: Amazon Linux 2
+      url: https://aws.amazon.com/amazon-linux-2/
+  tags: [ all, production, rpm, amazonlinux ]
+
+- name: amzn1
+  type: repository
+  desc: Amazon Linux 1
+  family: fedora
+  minpackages: 2000
+  sources:
+    - name: updates
+      fetcher: RepodataSqliteFetcher
+      parser: RepodataSqliteParser
+      url: http://repo.us-west-2.amazonaws.com/latest/updates/SRPMS/mirror.list
+    - name: main
```

Comment:
Both sources can be described by one entry, with `- name: [main, updates]`, `url: 'http://repo.us-west-2.amazonaws.com/latest/{source}/SRPMS/mirror.list'` and added `subrepo: '{source}'` see rpm/centos.yaml for example.